modules/client-configuration/pages/autoyast-example.adoc

```diff
@@ -1,103 +1,20 @@
 [[autoyast-example]]
 = AutoYast Example File
 
-{suse} provides templates of AutoYaST profiles in the https://github.com/SUSE/manager-build-profiles[SUSE/manager-build-profiles] public GitHub repository.
+{suse} provides templates of various AutoYaST profiles in the https://github.com/SUSE/manager-build-profiles/tree/master/AutoYaST[SUSE/manager-build-profiles/AutoYaST] public GitHub repository.
 
+Each script requires you to set some variables before you use it.
```

Comment:
```suggestion
Each profile requires you to set some variables before you use it.
```
I think we'd better do not switch (descriptive) terms from one sentence to the next.  Or we must say why we now want to call the profiles scripts.  This is just a proposal and in general we can address this later (we probably must check the complete autoinstallation section).  I always forgot to create an issue about this.